### ISSUE TITLE
[Bug]: Possible Issue Deleting Comment: [Spammer ID] | [Full URL]

### ISSUE THREAD

**Issue by Foxy6670** [ThioJoe/YT-Spammer-Purge#406]
### Duplicate Issues

- [X] There are **no existing posts relating to my problem**
- [ ] There are **existing posts relating to my problem, but the solution given, doesn't work for me.**

### What happened?

This is my third time attempting to get this running. Every single time I try to delete comments using this, it deletes only around 50 (There's ~800 to delete) and sometimes none. I've followed each step on the instructions exactly. I do not understand why it's failing for my channel. 

Thank you for any help in advance.
-foxo

P.S. Yes, I'm logged into the same account as my YouTube channel, and yes, I added myself to the created project.

### Release version

2.13.0

### Steps to reproduce

NOTE: I AM USING MANJARO AND PYTHON 3.10.1.

1. Follow the API setup step by step
2. Rename the client secrets file to "client_secrets.json" and add it to the same folder as the script
3. Authenticate, and run the script, and try to mass delete spam comments (850) from your own channel
If you did exactly what the instructions say, you should get this:
```
Comments Deleted! Will now verify each is gone.                          

                               
    (Note: You can disable deletion success checking in the config file, to save time and API quota)

Possible Issue Deleting Comment: [Comment ID] |  Check Here: [YouTube URL]
Possible Issue Deleting Comment: [Comment ID] |  Check Here: [YouTube URL]
Possible Issue Deleting Comment: [Comment ID] |  Check Here: [YouTube URL]
```
Or something similar. This message appeared around **800** times.

### What platform are you seeing this problem on?

Linux (can also be ChromeOS)

### Relevant log output

```shell
No crashes.
```

**Comment by ThioJoe** [comment]
Hm if you try older releases does it work for them?

**Comment by ThioJoe** [comment]
Ah crap I found the problem I think. Pretty sure it was this change: https://github.com/ThioJoe/YT-Spammer-Purge/commit/d9b09a5afc3930a1f8f68399fea86b1d12d30a22#diff-b76a2746f9e1b1f511aa9e62d7132810627e6602384a679887461c57acd9e04aR616

**Comment by ThioJoe** [comment]
Ok it _should_ be fixed with 2.13.1. Let me know if it works now

**Comment by ThioJoe** [comment]
Gonna assume this is fixed
